x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/foooter.tsx

```diff
@@ -59,45 +59,47 @@ const FlyoutFooterComponent: React.FC<Props> = ({
             </EuiButtonEmpty>
           )}
         </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiFlexGroup justifyContent="spaceBetween">
-            <>
-              {onTestConnector && (
+        {buttonType === 'back' ? (
```

Comment:
nit: would it be clearer to pass a prop like `hasConnectorTypeSelected` into the footer and determine the button type and whether to show the save buttons based on that flag instead of on whether the button type is `back`?

Reply:
Thanks! You are right. I don't like it either even though I put it there hahhah.